Donation-platform/src/main/webapp/trending.html

```diff
@@ -4,10 +4,18 @@
     <meta charset="UTF-8">
     <title>Trending Page</title>
     <script src="trendingScript.js"></script>
+    <link rel="stylesheet" href="trendingStyle.css">
```

Comment:
Does trendingStyle.css exist yet? I'm not seeing it.

Reply:
I have it locally, it has a bit of stuff in it - I'll add it in the next commit.

Reply:
This is ideally not a good practice since it will break once you submit and if we try to bring the webapp because of this missing file. But for now, LGTM since we want to iterate faster. But let's try to avoid it in future.